mixer/adapter/kubernetesenv/config/adapter.kubernetesenv.config.pb.html

```diff
@@ -63,6 +63,20 @@ <h2 id="Params">Params</h2>
 
 <p>Default: 5 minutes</p>
 
+</td>
+</tr>
+<tr id="Params-cluster_registries_namespace">
```

Comment:
a minor nit on naming here.  It is a single cluster registry.  Not a a number of Registries.  If you change it you need to change it in a number of places.

Reply:
Pilot uses the same name, so kept the same. If others comment then we should change in both places.